### PR TITLE
Fix issue #52: to convert to become Go Module

### PR DESCRIPTION
**Pull request by jim-wang-intel**
This PR is to address the issue #52  (Convert to Go Module) for Edinburgh release.
- VERSION number bumped up to 0.5.0
- Remove glide.yaml as it is no longer used
- Use logging from go-mod-core-contracts instead of original edgex-go package
- go.mod created for this module

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>

### CODE REVIEW

**Comment by jim-wang-intel** [comment]
@tingyuz @lenny-intel @tsconn23 please review.

Thanks, Jim Wang

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.2.1
+0.5.0
```
Comment (anonymouse64) [inline VERSION]: I think the security-services version going forward is supposed to be 1.0.0 for Edinburgh/master

Comment (jim-wang-intel) [reply VERSION]: ok will change it

**Comment by jim-wang-intel** [comment]
@tingyuz  so for the final version change: should we open an issue and fix toward it so that there is record on track?

**Pull request merged**